### PR TITLE
state store query API: remove 'value' prefix from key names

### PR DESCRIPTION
**Pull request by dmitsh**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Remove 'value' prefix from key names in statestore query API

## Issue reference

#2191

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md [hunk 1]
```diff
@@ -93,11 +95,11 @@ This is the [query](../query-api-examples/query1.json):
 ```json
 {
     "filter": {
-        "EQ": { "value.state": "CA" }
+        "EQ": { ".state": "CA" }
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md]: Please update the notation here to remove the dot
```suggestion
        "EQ": { "state": "CA" }
```

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md [hunk 2]
```diff
@@ -190,15 +192,15 @@ This is the [query](../query-api-examples/query2.json):
 ```json
 {
     "filter": {
-        "IN": { "value.person.org": [ "Dev Ops", "Hardware" ] }
+        "IN": { ".person.org": [ "Dev Ops", "Hardware" ] }
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md]:
```suggestion
        "IN": { "person.org": [ "Dev Ops", "Hardware" ] }
```

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md [hunk 3]
```diff
@@ -232,27 +234,27 @@ This is the [query](../query-api-examples/query3.json):
     "filter": {
         "OR": [
             {
-                "EQ": { "value.person.org": "Dev Ops" }
+                "EQ": { ".person.org": "Dev Ops" }
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md]:
```suggestion
                "EQ": { "person.org": "Dev Ops" }
```

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md [hunk 4]
```diff
@@ -232,27 +234,27 @@ This is the [query](../query-api-examples/query3.json):
...
             },
             {
                 "AND": [
                     {
-                        "EQ": { "value.person.org": "Finance" }
+                        "EQ": { ".person.org": "Finance" }
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md]:
```suggestion
                        "EQ": { "person.org": "Finance" }
```

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md [hunk 5]
```diff
@@ -232,27 +234,27 @@ This is the [query](../query-api-examples/query3.json):
...
                     },
                     {
-                        "IN": { "value.state": [ "CA", "WA" ] }
+                        "IN": { ".state": [ "CA", "WA" ] }
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md]:
```suggestion
                        "IN": { "state": [ "CA", "WA" ] }
```

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md [hunk 6]
```diff
@@ -232,27 +234,27 @@ This is the [query](../query-api-examples/query3.json):
...
                     }
                 ]
             }
         ]
     },
     "sort": [
         {
-            "key": "value.state",
+            "key": ".state",
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md]:
```suggestion
            "key": "state",
```

File: daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md [hunk 7]
```diff
@@ -232,27 +234,27 @@ This is the [query](../query-api-examples/query3.json):
...
             "order": "DESC"
         },
         {
-            "key": "value.person.id"
+            "key": ".person.id"
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/state-management/howto-state-query-api.md]:
```suggestion
            "key": "person.id"
```

**Comment by msfussell** [comment]
@dmitsh - Will leave you to fix this to remove the dot notation

**Comment by dmitsh** [comment]
> @dmitsh - Will leave you to fix this to remove the dot notation

done + some extra text

File: daprdocs/content/en/reference/api/state_api.md [hunk 8]
```diff
@@ -336,27 +336,27 @@ curl http://localhost:3500/v1.0-alpha1/state/myStore/query \
         "filter": {
           "OR": [
             {
-              "EQ": { "value.person.org": "Dev Ops" }
+              "EQ": { ".person.org": "Dev Ops" }
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/state_api.md]:
```suggestion
              "EQ": { "person.org": "Dev Ops" }
```

File: daprdocs/content/en/reference/api/state_api.md [hunk 9]
```diff
@@ -336,27 +336,27 @@ curl http://localhost:3500/v1.0-alpha1/state/myStore/query \
...
             },
             {
               "AND": [
                 {
-                  "EQ": { "value.person.org": "Finance" }
+                  "EQ": { ".person.org": "Finance" }
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/state_api.md]:
```suggestion
                  "EQ": { "person.org": "Finance" }
```

File: daprdocs/content/en/reference/api/state_api.md [hunk 10]
```diff
@@ -336,27 +336,27 @@ curl http://localhost:3500/v1.0-alpha1/state/myStore/query \
...
                 },
                 {
-                  "IN": { "value.state": [ "CA", "WA" ] }
+                  "IN": { ".state": [ "CA", "WA" ] }
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/state_api.md]:
```suggestion
                  "IN": { "state": [ "CA", "WA" ] }
```

File: daprdocs/content/en/reference/api/state_api.md [hunk 11]
```diff
@@ -336,27 +336,27 @@ curl http://localhost:3500/v1.0-alpha1/state/myStore/query \
...
                 }
               ]
             }
           ]
         },
         "sort": [
           {
-            "key": "value.state",
+            "key": ".state",
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/state_api.md]:
```suggestion
            "key": "state",
```

File: daprdocs/content/en/reference/api/state_api.md [hunk 12]
```diff
@@ -336,27 +336,27 @@ curl http://localhost:3500/v1.0-alpha1/state/myStore/query \
...
             "order": "DESC"
           },
           {
-            "key": "value.person.id"
+            "key": ".person.id"
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/state_api.md]:
```suggestion
            "key": "person.id"
```

**Review by msfussell — COMMENTED** [review]
Still a few more dots to check

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**